raspberry/bzzz/sensors/gnss.py

```diff
@@ -4,40 +4,221 @@
 import time
 import datetime
 from .data_logger import DataLogger
-from .filters import MedianFilter
+from .filters import *
 
-def _deg_min_sec_to_decimal(degrees, minutes, direction):
-    """
-    Convert degrees-minutes-seconds from GNGLL to decimal
-    :param degrees: degrees
-    :param minutes: minutes
-    :param direction: direction (can be one of 'N', 'W', 'E', 'S')
-    :return: decimal representation of GPS coordinates
-    """
-    decimal_degrees = degrees + minutes / 60
-    if direction in ['S', 'W']:  # need to be denoted with a negative value
-        decimal_degrees *= -1
-    return decimal_degrees
 
 class Gnss:
     """
     GNSS Module
 
     This class is used to interface the GNSS Module
     """
+
+    #RELPOSNED and PVT
+    # Taken from https://github.com/mnltake/readF9P_UBX
+    def __readUBX(self, readbytes):
```

Comment:
Is `self` used in this method? If not, it can be a function and moved outside the class.